cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/services/FileImporter.java

```diff
@@ -73,7 +75,15 @@ public Crac importCracFromJson(String cracUrl) throws IOException {
     }
 
     public ZonalData<Scalable> importGlsk(String glskUrl, Network network) throws IOException {
-        return GlskDocumentImporters.importGlsk(urlValidationService.openUrlStream(glskUrl)).getZonalScalable(network);
+        ZonalData<Scalable> zonalScalable = GlskDocumentImporters.importGlsk(urlValidationService.openUrlStream(glskUrl)).getZonalScalable(network);
+        Arrays.stream(CseCountry.values()).forEach(val -> checkCseCountryInGlsk(zonalScalable, val));
+        return zonalScalable;
+    }
+
+    private void checkCseCountryInGlsk(ZonalData<Scalable> zonalScalable, CseCountry val) {
```

Comment:
I would prefer if the check was in the DIchotomyRunner for example, so that the FileImporter has no business logic. It just imports file, then we can do business checks in class that are more business related, like DichotomyRunner.